### PR TITLE
Distribution of species overlays by districts and countries

### PR DESCRIPTION
**Pull request by billli0**
### Issue #:
https://linear.app/bes/issue/SOL-81/regional-map-overlay-country
https://linear.app/bes/issue/SOL-80/map-overlay-district

### CODE REVIEW

File: packages/database/src/migrations/20210824231816-DistributionOfSpeciesPacmoss-modifies-data.js [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+  ['Aedes', 'Ae_Polynesiensis'],
+  ['Aedes', 'Ae_Rotumae'],
+  ['Aedes', 'Ae_Scutellaris'],
+  ['Aedes', 'Ae_Vigilax'],
+  ['Culex', 'Cx_Annulirostris'],
+  ['Culex', 'Cx_Quinquefasciatus'],
+  ['Culex', 'Cx_Sitiens'],
+  ['Mansonia', 'Mn_Uniformis'],
+];
+const HIERARCHIES = ['Country', 'District'];
```
Comment (nick-beyondessential) [inline packages/database/src/migrations/20210824231816-DistributionOfSpeciesPacmoss-modifies-data.js:37]:
```suggestion
const HIERARCHYLEVELS = ['Country', 'District'];
```
Optional naming suggestion just to make it a bit clearer.

File: packages/report-server/src/routes/helpers.ts [hunk 2]
```diff
@@ -39,7 +39,6 @@ export const getRequestedOrgUnitObjects = async (
     throw new Error(`No entities found with codes ${orgUnitCodesInArray}`);
   }
 
-  // If entity is a project
```
Comment (nick-beyondessential) [inline packages/report-server/src/routes/helpers.ts:42]: This seems like a weird change was it from the conflict?

**Review by nick-beyondessential — APPROVED** [review]
LGTM

File: packages/report-server/src/routes/helpers.ts (hunk 2 above)
Comment (billli0) [reply packages/report-server/src/routes/helpers.ts:42]: I delete it for a purpose since the code can explain itself.

Comment (nick-beyondessential) [reply packages/report-server/src/routes/helpers.ts:42]: Good point :)

**Pull request merged**